src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -28,6 +28,7 @@ struct event_entry_t {
   uint32_t time_to_live = DEFAULT_GLOBAL_VALUE;
   uint32_t max_retries = DEFAULT_GLOBAL_VALUE;
   uint32_t retry_sleep_duration = DEFAULT_GLOBAL_VALUE;
+  bool needs_migration = true;
```

Comment:
can we use `creation_time == ceph::coarse_real_clock::zero()` as an indication that mogration is needed, instead of adding a new data member?

Reply:
done